README.md

```diff
@@ -129,6 +129,66 @@ env "gorm" {
 }
 ```
 
+### Extra Features
+
+#### Views
+
+> Note: Views are available for logged-in users, run `atlas login` if you haven't already. To learn more about logged-in features for Atlas, visit [Feature Availability](https://atlasgo.io/features#database-features).
+
+To define a Go struct as a database `VIEW`, implement the `ViewDef` method as follow:
+```go
+// User is a regular gorm.Model stored in the "users" table.
+type User struct {
+  gorm.Model
+  Name string
+  Age  int
+}
+
+// WorkingAgedUsers is mapped to the VIEW definition below.
+type WorkingAgedUsers struct {
+  Name string
+  Age  int
+}
+
+func (WorkingAgedUsers) ViewDef(dialect string) []gormschema.ViewOption {
+  return []gormschema.ViewOption{
+    gormschema.BuildStmt(func(db *gorm.DB) *gorm.DB {
+      return db.Model(&User{}).Where("age BETWEEN 18 AND 65").Select("name, age")
+    }),
+  }
+}
+```
+In order to pass a plain `CREATE VIEW` statement, use the `CreateStmt` as follows:
+```go
+type BotlTracker struct {
+  ID   uint
+  Name string
+}
+
+func (BotlTracker) ViewDef(dialect string) []gormschema.ViewOption {
+  var stmt string
+  switch dialect {
+  case "mysql":
+    stmt = "CREATE VIEW botl_trackers AS SELECT id, name FROM pets WHERE name LIKE ?"
+  }
+  return []gormschema.ViewOption{
+    gormschema.CreateStmt(stmt, "botl%"),
```

Comment:
Parameterized statements are not part of the DDL definition. Also, since we don't deal with untrusted user inputs, there's no need to parametrized statements at all. 
```suggestion
    gormschema.CreateStmt(stmt),
```